package-lock.json

```diff
@@ -1,9921 +1,7 @@
 {
```

Comment:
What happened here? I don't think we want to remove this lock data 😄

Reply:
Some sort of accident... I actually do not know o_O - Reverted